tests/client-dbus/check.py

```diff
@@ -5,23 +5,19 @@
 import sys
 
 arg_map = {
-   "src/stratisd_client_dbus" : [
-      "--reports=no",
-      "--disable=I",
-      "--disable=bad-continuation",
-      "--disable=invalid-name",
-      "--msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'"
-   ],
-   "tests" : [
-      "--reports=no",
-      "--disable=I",
-      "--disable=bad-continuation",
-      "--disable=duplicate-code",
-      "--disable=invalid-name",
-      "--msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'"
-   ]
+    "src/stratisd_client_dbus": [
+        "--reports=no", "--disable=I", "--disable=bad-continuation",
```

Comment:
You can remove bad-continuation disable here.

Reply:
Is your comment related to the PEP8 formatting?  If this is an unrelated change I'll open a new PR.

Reply:
Yes. By adding the PEP8 formatting you make all the "bad-continuation" lint errors go away.

Reply:
Ok.